Server/src/main/resources/static/js/editor.js

```diff
@@ -183,81 +230,25 @@ function getOperandType(text, result) {
         return OPERAND_INVALID;
     }
 
-    //Check IMM
-    if (!isNaN(Number(text)) && Number(text) === Math.floor(Number(text)) && text.indexOf("o") === -1
-        && text.indexOf("0e") !== 0) {
+    if (isImmediateOp(text, result)) {
```

Comment:
`result` is never defined here, this raises a `ReferenceError`

Reply:
isn't result a function attribute (line 226)